src/components/RegisterForm/BarCodeScanner.vue

```diff
@@ -26,16 +23,25 @@ const emit = defineEmits<{
 const toastStore = useToast()
 
 const codeReader = new BrowserBarcodeReader()
-const inputs = ref<VideoInputDevice[]>([])
-const selectInput = ref<VideoInputDevice>()
 const videoEle = shallowRef<HTMLVideoElement>()
+const deviceOptions = ref<DeviceOption[]>([])
```

Comment:
```suggestion
const deviceOptions = computed<DeviceOption[]>(() => inputs.map(v => ({
          key: v?.deviceId ?? '',
          label: v?.label ?? 'no device name'
        })))
```
かな？
`computed`を使うと更新のし忘れがなくなるからこうしたいです
現状`VideoInputDevice[]`は`initialize`でしか更新されてませんが今後ほかのところで更新することになるかもしれないので